crates/wasmi/src/store.rs

```diff
@@ -80,7 +97,8 @@ pub struct Store<T> {
     /// Stored host function trampolines.
     trampolines: Arena<TrampolineIdx, TrampolineEntity<T>>,
     /// User provided host data owned by the [`Store`].
-    data: T,
+    pub(crate) data: T,
+    pub(crate) limiter: Option<ResourceLimiterQuery<T>>,
```

Comment:
With the aforementioned getters for `ResourceLimiterRef` we can get rid of those `pub(crate)`.

Reply:
Will do.